.github/workflows/test.yaml

```diff
@@ -332,7 +335,14 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
+      - name: use node 12
+        if: ${{ matrix.platform != 'firebase-functions' }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
       - name: use node 10
+        if: ${{ matrix.platform == 'firebase-functions' }}
```

Comment:
?

Reply:
I discussed this with @divyenduz; we need to use node 12 for redwood so we just upgrade all to node 12, but firebase currently only runs on node 10 so we make an exception for firebase.

Reply:
A comment in the code would be welcome as this context is impossible to get for other people otherwise.

Reply:
https://github.com/prisma/e2e-tests/commit/3ff6662176d98c2a4ce7b7275283288be9a9c764

Reply:
Created a tracking issue and linked it via https://github.com/prisma/e2e-tests/commit/3ff6662176d98c2a4ce7b7275283288be9a9c764